tls/vibe/stream/openssl.d

```diff
@@ -336,6 +338,7 @@ final class OpenSSLStream : TLSStream {
 	size_t read(scope ubyte[] dst, IOMode mode)
 	{
 		size_t nbytes = 0;
+		if(m_tls == null) return 0;
```

Comment:
This one should throw an exception instead (e.g. `new Exception("Reading from closed TLS stream")`).